docs/getting-started/02-create-namespace.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Create a Namespace
+type: Getting Started
+---
+
+Almost all operations in these guides are performed using [Namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)-scoped resources, so let's start by creating a dedicated `orders-service` Namespace for them.
+
+## Reference
+
+This and all other guides demonstrate steps you can perform from both the terminal (kubectl) and UI. Read about the [Console](/components/console) through which you can visually and [securely](/components/security/) administer Kyma functionalities and manage the basic Kubernetes resources.
```

Comment:
```suggestion
This and all other guides demonstrate steps you can perform from both the terminal (kubectl) and the Console UI. Read about the [Console](/components/console) through which you can graphically and [securely](/components/security/) administer Kyma functionalities and manage the basic Kubernetes resources.
```
**Reason:** 
In the IT context, we often talk about `graphical user interface` (GUI). `visually` seems to me to be related to vision, things you can see, which includes text.
Also, if you compare Google [hits for `visually administer`](https://www.google.com/search?ei=doVHX7LVAYrsrgTooJPwDg&q=%22visually+administer%22&oq=%22visually+administer%22) and [hits for `graphically administer`](https://www.google.com/search?ei=vYVHX7alKKL2qwHnm4mgCw&q=%22graphically+administer%22), the latter are much more numerous and appear in more formal contexts. 
 I believe the word `graphically` is more suitable here. See the examples from the _Cambridge English Corpus_ at the bottom of [this](https://dictionary.cambridge.org/dictionary/english/graphically) page.

Reply:
I also changed `steps you can perform from ` -> `steps you can perform in `

Reply:
Cool :) 👍